jsk_2016_01_baxter_apc/euslisp/main-stow.l

```diff
@@ -115,23 +115,42 @@
               (ros::ros-info "[main] ~a, place object in bin ~a" arm target-bin)
               (send *ri* :move-arm-body->bin arm target-bin :scale 5.0)
               (send *ri* :wait-interpolation)
-              (setq hcv (send *ri* :get-val '_hard-coded-variables))
-              (if (and object-length (> 0.20 object-length))
-                (setq entrance-x-list (list -200 -150 -100 -50 0 150))
-                (setq entrance-x-list (list -200 -150 -100 -50 0 150 200)))
+              (setq offset-avoid-bin-top
+                    (gethash :offset-avoid-bin-top (send *ri* :get-val '_hard-coded-variables)))
+              (if (and object-length (> object-length 0.2))
+                (progn
+                  (setq entrance-x-list (list -200 -150 -100 -50 0 150 200))
+                  (setq offset-avoid-bin-top 50)
```

Comment:
Is this offset ok for Bin d, e, f, g, h, i?
If ok, don't worry about this comment.